README.md

```diff
@@ -2,4 +2,7 @@
 # Mandrill service integration for Magento 2.
 
 If you are running Magento 2.1.x use the 3.0 branch.
-If you are running Magento 2.2.x use the 3.1 branch.
+
+If you are running Magento 2.2.0 to 2.2.7 use the 3.1 branch.
+
+**If you are running Magento 2.2.8 or greater use [ebizmarts/magento2-mandrill-smtp](https://github.com/ebizmarts/magento2-mandrill-smtp)**
```

Comment:
Wow. Really?
What is benefit of using SMTP instead of Mandrill API?

Reply:
No pros and no cons, simply a different approach

Reply:
Yeah, but why you decided to develop separate module instead of this one?
There are a lot of modules that works with smtp, for instance https://github.com/magepal/magento2-gmail-smtp-app
And no alternative modules that works using api.
From this PR it looks that you’re stopping support of this module. Am I correct?